frontend/packages/dev-console/src/components/import/devfile/devfile-utils.ts

```diff
@@ -0,0 +1,95 @@
+import * as _ from 'lodash';
+import { GitProvider, getGitService } from '@console/git-service/src';
+import { coFetch } from '@console/internal/co-fetch';
+import { K8sResourceKind } from '@console/internal/module/k8s';
+import { safeYAMLToJS, safeJSToYAML } from '@console/shared/src/utils/yaml';
+import { DevfileComponent } from './devfile-types';
+
+export const suffixSlash = (val: string) => (val.endsWith('/') ? val : `${val}/`);
+
+export const prefixDotSlash = (val) => (val.startsWith('/') ? `.${val}` : val);
+
+export const getResourceContent = async (
+  resourceURI: string,
```

Comment:
Add name for error messages below

```suggestion
export const getResourceContent = async (
  componentName: string,
  resourceURI: string,
```